lib/application.ts

```diff
@@ -196,6 +200,43 @@ export class Application {
                 return this.scheduler.perform(`Resolving "${targetProcess.name}"`, (): Promise<frida.Process> => {
                     return device.getProcess(targetProcess.name);
                 });
+            case "by-gating":
+                return await this.scheduler.perform(`Waiting for "${targetProcess.name}"`, (): Promise<frida.Process> => {
+                    return new Promise((resolve, reject) => {
+                        this.onSpawnGatingDisabled = fatReject;
+
+                        const onSpawnAdded: frida.SpawnAddedHandler = async (spawn) => {
+                            const { identifier, pid } = spawn;
+                            
+                            const processes = await device.enumerateProcesses();
+                            const proc = processes.find((p: frida.Process) => p.pid === pid);
```

Comment:
```suggestion
                            const proc = processes.find(p => p.pid === pid);
```

(To make it more readable as we know in this context what `p` is.)